src/pages/NotFound/__tests__/component.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NotFound from '../component';
+
+test('NotFound component matches snapshot', () => {
```

Comment:
Please, add describe block and fix messages according our last discussion

Reply:
Done